github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml

```diff
@@ -795,130 +795,130 @@ spec:
     app: alertmanager
     alertmanager: prometheus-stack-kube-prom-alertmanager
   type: "ClusterIP"
+# ---
```

Comment:
I think it would be better to remove this dead code

Reply:
Agree!